src/style.css

```diff
@@ -0,0 +1,18 @@
+* {
```

Comment:
you only need to change default `margin` on `body`, no need to add this to all elements

Reply:
Thank yo for review, i corrected mistakes